prettier.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = { // Prettier defaults
+  singleQuote: true, // singleQuote: true
```

Comment:
Why is everything written in comment as well?

Reply:
Because configuration files where you have to spend 30 minutes looking up the defaults and comparing them line by line are completely useless. The point of using a `.js` file is that you can add comments 😀

Reply:
Thanks! Understood now. I am a new to open source. I was checking out things. I hope to read this repo thoroughly and gain some good practices.
Do you know any React + TypeScript projects I can contribute to?

Reply:
@ThePiyushAggarwal There's a TypeScript version of https://github.com/timlrx/tailwind-nextjs-starter-blog you could look into.

Reply:
Thanks a lot! Your profile looks great.
I am going to check out every single thing.
I will catch up with you on twitter sometime.
😄